test/Common/Thermodynamics/runtests.jl

```diff
@@ -430,6 +430,7 @@ end
             getproperty.(q_pt, :liq) .+ getproperty.(q_pt, :ice),
         )
         @test all(has_condensate.(q_dry) .== false)
+        @test all(specific_enthalpy.(ts) .≈ e_int .+ gas_constant_air.(ts) .* air_temperature.(ts))
```

Comment:
You might also want to test that everything composes correctly, i.e., `total_specific_enthalpy = specific_enthalpy + e_kin`, `moist_static_energy = specific_enthalpy + geopotential` and `moist_static_energy = total_specific_enthalpy - e_kin`.

Reply:
Right now, we only ever test single velocity/momentum/kinetic energy values, since they're not thermodynamic quantities. Is there a good profile we can use for velocity, momentum, or kinetic energy? I added the `e_pot` profile (`grav * z`), so now that's tested for a range of values rather than a single value.